docs/data/galexie/admin_guide/installing.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Installing
+sidebar_position: 30
+---
+
+# Installing
```

Comment:
Should we add a link to the github readme instructions for installing/building locally?

Reply:
Good point. I don't think we currently have instructions for building it locally but it might be worth adding them.